cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/ZooKeeperRoller.java

```diff
@@ -128,14 +133,76 @@ public Future<Void> maybeRollingUpdate(Reconciliation reconciliation, Labels sel
      * @return a Future which completes when the given (possibly recreated) pod is ready.
      */
     Future<Void> restartPod(Reconciliation reconciliation, String podName, List<String> reasons) {
-        long pollingIntervalMs = 1_000;
-
         LOGGER.infoCr(reconciliation, "Rolling pod {} due to {}", podName, reasons);
         return podOperator.getAsync(reconciliation.namespace(), podName)
                 .compose(pod -> podOperator.restart(reconciliation, pod, operationTimeoutMs))
                 .compose(ignore -> {
                     LOGGER.debugCr(reconciliation, "Waiting for readiness of pod {}", podName);
-                    return podOperator.readiness(reconciliation, reconciliation.namespace(), podName, pollingIntervalMs, operationTimeoutMs);
+                    return podOperator.readiness(reconciliation, reconciliation.namespace(), podName, READINESS_POLLING_INTERVAL_MS, operationTimeoutMs);
                 });
     }
+
+
```

Comment:
Do we need the two empty lines everwhere? 😂

Reply:
No :( codestyle from my last job sorry, will fix.